README.md

```diff
@@ -35,32 +35,52 @@ Understanding how humans leverage semantic knowledge to navigate unfamiliar envi
 
 ## :hammer_and_wrench: Installation
 
-### Getting Started
+### Getting Started: Environment
 
 Create the conda environment:
 ```bash
 conda_env_name=vlfm
 conda create -n $conda_env_name python=3.9 -y &&
 conda activate $conda_env_name
 ```
+
+Check compiled cuda version/available cuda version and install the closest available corresponding torch. *This will be important for successful GroundingDINO installation*
+
 If you are using habitat and are doing simulation experiments, install this repo into your env with the following:
 ```bash
 pip install -e .[habitat]
 ```
+
 If you are using the Spot robot, install this repo into your env with the following:
 ```bash
 pip install -e .[reality]
 ```
-Install all the dependencies:
+
+Clone the below dependencies into the root directory of this repo:
 ```bash
-git clone git@github.com:IDEA-Research/GroundingDINO.git
-git clone git@github.com:WongKinYiu/yolov7.git  # if using YOLOv7
+git clone https://github.com/IDEA-Research/GroundingDINO.git
+git clone https://github.com/WongKinYiu/yolov7.git  # if using YOLOv7
+git clone https://github.com/facebookresearch/habitat-lab
+git clone https://github.com/naokiyokoyama/bd_spot_wrapper.git
 ```
 Follow the original install directions for GroundingDINO, which can be found here: https://github.com/IDEA-Research/GroundingDINO.
 
 Nothing needs to be done for YOLOv7, but it needs to be cloned into the repo.
 
-### Installing GroundingDINO (Only if using conda-installed CUDA)
+Follow the original install directions for Habitat-lab, which can be found here: https://github.com/facebookresearch/habitat-lab.
+
+Follow the original install directions for bd spot wrapper, which can be found here: https://github.com/naokiyokoyama/bd_spot_wrapper.git.
+
+The following pip install's will also be required:
+
+```bash
+pip install hydra-core --upgrade
```

Comment:
It would be good to move these to pyproject.toml or the Dockerfile as much as possible.

Reply:
True! If I get the chance will move it there nonetheless thought it was useful information incase the requirements in pyproject.toml wasn't enough to get your environment ready

Reply:
Both habitat-lab and bd_spot_wrapper have been in the toml file already